src/app/helptext/storage/volumes/volume-list.ts

```diff
@@ -125,6 +125,18 @@ encryption_options_dialog: {
     confirm_checkbox: T('Confirm'),
     save_button: T('Save')
 },
+pool_options_dialog: {
+    dialog_title: T('Edit Pool Options for '),
+    autotrim_placeholder:T('Auto TRIM'),
+    autotrim_tooltip: T(''),
```

Comment:
Help text suggestion:
`Enable for TrueNAS to periodically review data blocks and identify empty blocks of obsolete blocks that can be deleted. Unset to incorporate day block overwrites when a device write is started (default).